src/lib.rs

```diff
@@ -296,3 +317,22 @@ pub async fn run(config: Config) -> Result<()> {
         }
     }
 }
+
+pub async fn image_present_locally(images: Images, search_for_image: String) -> bool {
+    match images.list(&Default::default()).await {
+        Ok(images) => {
+            for image in images {
+                for repo_tag in image.repo_tags {
+                    for tag in repo_tag {
```

Comment:
It's possible to express this in one line in Rust via `.iter()` and `.map()` versus Go -- we can merge as-is and then refactor it as a follow-up.